### PR TITLE
Fit runtime field on line

### PR DESCRIPTION
**Pull request by nik9000**
This shrinks a runtime field definition so that it fits on the screen
without scrolling. It also converts the doc into a test so we can be
sure it continues to work.

Relates to #69291

### CODE REVIEW

File: docs/reference/search/search.asciidoc [hunk 1]
```diff
@@ -452,15 +452,36 @@ script has access to the entire context of a document, including the original
 Your script must include `emit` to return calculated values. For
 example:
 +
-[source,js]
+[source,console]
```
Comment (jrodewig) [inline docs/reference/search/search.asciidoc:455]: Quick drive-by:

This does add some of the console chrome to this snippet:
Screen Shot 2021-04-08 at 12 03 36 PM

This doesn't mangle the line in copy as cURL, but it's kinda odd... you can't exactly run this in as is in curl or Kibana console. Still probably worth it for the testing coverage tho.

Comment (nik9000) [reply docs/reference/search/search.asciidoc:455]: Hmmm - I can probably avoid the chrome.

Also, on my screen it fits on the screen. no scroll bar! Hmmm.

Comment (jrodewig) [reply docs/reference/search/search.asciidoc:455]: The scroll bar is likely the result of my window size. 😄

Comment (jrodewig) [reply docs/reference/search/search.asciidoc:455]: +1 on removing that chrome. Let me know how you do it! It could be useful for other work too. For example, I want to switch several of our JS snippets for ingest processors over to console snippets. This would let me just show the object rather than an entire request.

Comment (nik9000) [reply docs/reference/search/search.asciidoc:455]: image

Comment (nik9000) [reply docs/reference/search/search.asciidoc]: @jrodewig I ended up using the `include` trick on a commented out block.That skips the chrome and keeps the test.

Comment (jrodewig) [reply docs/reference/search/search.asciidoc]: Thanks @nik9000! Much more readable too!
